ci-operator/jobs/openshift/cincinnati/openshift-cincinnati-master-presubmits.yaml

```diff
@@ -180,23 +180,23 @@ presubmits:
     always_run: true
     branches:
     - master
-    cluster: build01
```

Comment:
This change and the others where the cluster is changed doesn't logically belong with the version bump

Reply:
These are autogenerated files